components/docs-chef-io/content/automate/servicenow_integration_reference.md

```diff
@@ -145,6 +145,38 @@ The **Field Maps** establishes a relationship between a field in an import set t
 
 {{< figure src="/images/automate/snow_integration_transform_map_scripts1.png" alt="Transform Map Scripts">}}
 
+## Discovery
+
+[ServiceNow](https://www.servicenow.com/) Discovery feature finds applications and devices on your network, and then updates the CMDB with the information it finds. It discovers both physical and logical components, including virtual machines, servers, storage, databases, applications, and more.
```

Comment:
```suggestion
[ServiceNow's](https://www.servicenow.com/) Discovery feature finds applications and devices on your network, and then updates the ServiceNow's Configuration Management Database (CMDB) with the information it finds. It discovers both physical and logical components, including virtual machines, servers, storage, databases, applications, and more.
```

Reply:
Do I have to expand it? As the parent page has it - short and full form ?

Reply:
It's the first instance of CMDB on this page, so I think it makes sense to expand it.